v2/devdoc/log_sink_etw_requirements.md

```diff
@@ -30,7 +30,15 @@ Note: No other APIs (`deinit`, `log`) should be called while `init` executes.
 
 **SRS_LOG_SINK_ETW_01_008: [** `log_sink_etw.init` shall emit a `LOG_LEVEL_INFO` event as a self test, printing the fact that the provider was registered and from which executable (as obtained by calling `GetModuleFileNameA`). **]**
 
-**SRS_LOG_SINK_ETW_01_084: [** `log_sink_etw.init` shall use as provider GUID `DAD29F36-0A48-4DEF-9D50-8EF9036B92B4`. **]**
+**SRS_LOG_SINK_ETW_01_094: [** If `LOG_SINK_ETW_PROVIDER_GUID` is defined, its value should be used directly to initialize the provider GUID. **]**
+
+Note that the format for the contents of `LOG_SINK_ETW_PROVIDER_GUID` is the the format used to initialize a `GUID` struct. Example:
+
+```c
+(0xDAD29F36, 0x0A48, 0x4DEF, 0x9D, 0x50, 0x8E, 0xF9, 0x03, 0x6B, 0x92, 0xB4)
```

Comment:
>( [](http://example.com/codeflow?start=0&length=1)

shouldn't this be `{`?

When can we initialize structs with `(`?

Reply:
Eeeeeh, it is a tracelogging construct. Well then - consider adding the fact that `(` is really needed (here and in CMake)

Reply:
The fact that () is needed has been abstracted by a cmake macro now, so it is easier to use.
Nevertheless, I added here the comment that () are needed, since this is still relevant for the #define.